arm-resources/locks/2015-01-01/swagger/locks.json

```diff
@@ -547,9 +588,6 @@
             "type": "string",
```

Comment:
This has been there for quite a while.  @vivsriaus do you have an idea about this?